src/main/kotlin/com/lykke/trade/volume/monitoring/service/notification/SentNotificationsCacheImpl.kt

```diff
@@ -0,0 +1,34 @@
+package com.lykke.trade.volume.monitoring.service.notification
+
+import com.lykke.trade.volume.monitoring.service.config.Config
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Component
+import java.util.*
+import java.util.concurrent.ConcurrentHashMap
+
+@Component
+class SentNotificationsCacheImpl(config: Config): SentNotificationsCache {
+    private val notificationConfig = config.tradeVolumeConfig.notificationsConfig
+    private val timestampByNotificationKey = ConcurrentHashMap<String, Date>()
+
+    override fun add(clientId: String, assetId: String) {
+        timestampByNotificationKey[getKey(clientId, assetId)] = Date()
+    }
+
+    override fun isSent(clientId: String, assetId: String): Boolean {
+        return timestampByNotificationKey[getKey(clientId, assetId)] != null
+    }
+
+    @Scheduled(fixedRateString = "#{Config.tradeVolumeConfig.notificationsConfig.throttlingPeriod}")
```

Comment:
`fixedRateString` should be get from another setting. `throttlingPeriod` is lifetime of record about sent notification. But we need frequency of cleaning here

Reply:
agreed to leave one parameter